termui/show_bug.go

```diff
@@ -603,6 +609,14 @@ func (sb *showBug) setTitle(g *gocui.Gui, v *gocui.View) error {
 	return setTitleWithEditor(sb.bug)
 }
 
+func (sb *showBug) toggleOpenClose(g *gocui.Gui, v *gocui.View) error {
+	if sb.bug.Snapshot().Status.String() == "open" {
```

Comment:
Could you use a switch/case with the constants (https://github.com/MichaelMure/git-bug/blob/master/bug/status.go#L12-L13) instead of a string ? In the future, a bug might have more status than just open/close, might as well prepare the code for that.